RevenueCat/Scripts/StoreTransaction.cs

```diff
@@ -17,13 +17,31 @@ public class StoreTransaction
          * Id associated with the transaction in RevenueCat.
          * </summary>
          */
+        [Obsolete("Deprecated, use TransactionIdentifier instead.", false)]
         public readonly string RevenueCatId;
+
+        /**
+         * <summary>
+         * Id associated with the transaction in RevenueCat.
+         * </summary>
+         */
+        public readonly string TransactionIdentifier;
+
```

Comment:
Ohh I see here's where the deprecation happened

Reply:
@vegaro had looked at these fields in the past, just tagging him to ensure this is what he had in mind for unifying